dev/view/settings.html

```diff
@@ -0,0 +1,8 @@
+<div class="tab-pane fade" id="tab-settings">
+    <!--include 'settings/navigation.html' -->
+    <div class="tab-content inner-tab-content">
+        <!--include 'settings/profile.html' -->
+        <!--include 'settings/customize.html' -->
+        <!--include 'settings/about.html' -->
+    </div>
+</div>
```

Comment:
Should be a part of `popup_dev.html`

Reply:
Do you mean a part of the settings.html file? as in the sub-templates should be in that file and not separated (still one depth)?